_rules/SC1-1-1-aria-describedby.md

```diff
@@ -104,6 +106,14 @@ The resulting assertion is as follows,
 | outcome     | Failed
 | description | None of the elements referenced by aria-describedby exists.
 
+### step3-undetermined
+
+| Property    | Value
+|-------------|----------
+| type        | TestResult
+| outcome     | Undetermined
```

Comment:
I think these should be `CannotTell` instead of `Undetermined`.

Reply:
We have here a problem because the inconsistence between two WC3 documents:

- ACT 1.0 establishes "Undetermined": https://www.w3.org/TR/act-rules-format/#output-outcome
- EARL 1.0 establishes "Cannot tell" coded as "earl:cantTell": https://www.w3.org/TR/EARL10-Schema/#OutcomeValue

It should be convenient that both documents editors unify this notation: ACT editor: Wilco and Maureen , EARL editor: Shadi.